src/styles/overview.scss

```diff
@@ -0,0 +1,36 @@
+.overview {
+  grid-template-columns: 1fr 1fr;
+  grid-auto-rows: 1fr;
+  gap: var(--amplify-space-xxl);
+
+  @media (max-width: 600px) {
+    grid-template-columns: 1fr;
+  }
+
+  &__link {
+    color: inherit;
+    text-decoration: none;
+
+    &:hover &__card__title {
+      color: var(--amplify-colors-teal-80);
+    }
+
+    &__card {
+      border-radius: var(--amplify-radii-small);
+      padding: var(--amplify-space-xl);
+      height: 100%;
+
+      &__title {
+        font-size: var(--amplify-font-sizes-large);
+      }
+
+      &__description {
+        /* Line clamp to hide text and show ellipses */
+        display: -webkit-inline-box;
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+        -webkit-line-clamp: 3;
```

Comment:
Setting the line clamp to 3 so that it shows an ellipses and truncates the description

https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

Reply:
This is a little wild to me because we're using -webkit-prefixed stuff, but I guess the caniuse charts check out 🤷‍♀️